lib/kafkalib/topic_test.go

```diff
@@ -117,7 +117,7 @@ func TestTopicConfig_String(t *testing.T) {
 
 func TestTopicConfig_Validate(t *testing.T) {
 	var tc TopicConfig
-	assert.False(t, tc.Valid(), tc.String())
+	assert.Error(t, tc.Validate(), tc.String())
```

Comment:
Use `assert.ErrorContains` with the expected messages?

Reply:
:+1: